tests/src/Nightwatch/Commands/fillEntityBrowser.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @file
+ * Automatically fill entity browser field widget in modal dialog.
+ *
+ * This provides a custom command, .fillEntityBrowser()
+ *
+ * TODO: Extend to accept list of entities to select.
+ *
+ * @param {string} fieldName
+ *   The filed name.
+ * @param {string} entityBrowserName
+ *   The entity browser name.
+ * @param {string} selectionMode
+ *   The entity browser selection mode.
+ *
+ * @return {object}
+ *   The 'browser' object.
+ */
+exports.command = function fillEntityBrowser(
+  fieldName,
+  entityBrowserName,
+  selectionMode
+) {
+  const browser = this;
+
+  const fieldIdPart = fieldName.replace(/[_[]/g, "-").replace(/]/g, "");
+  const entityBrowserNameIdPart = entityBrowserName
+    .replace(/[_[]/g, "-")
+    .replace(/]/g, "");
+
+  browser
+    .scrollInViewAndClick(
+      `//*[starts-with(@id, "edit-${fieldIdPart}-entity-browser-entity-browser-open-modal")]`
+    )
+    .waitForElementVisible(
+      `//*[@id="entity_browser_iframe_${entityBrowserName}"]`,
+      10000
+    )
+    .frame(`entity_browser_iframe_${entityBrowserName}`)
+    .waitForElementVisible(
```

Comment:
When I run this test locally, it times out at this command, When running it a second time it just works. Strange.